Tests.Unit.Presentation.Web/Services/OrganizationServiceTest.cs

```diff
@@ -816,6 +817,23 @@ public void Can_Get_Permissions_Returns_Not_Found()
             Assert.Equal(OperationFailure.NotFound, result.Error.FailureType);
         }
 
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin, true)]
+        [InlineData(OrganizationRole.GlobalAdmin, false)]
+        [InlineData(OrganizationRole.LocalAdmin,  true)]
+        [InlineData(OrganizationRole.LocalAdmin, false)]
+        [InlineData(OrganizationRole.User,  true)]
+        [InlineData(OrganizationRole.User, false)]
+        public void HasRoleWorks(OrganizationRole roleBeingAsked, bool isActualRole)
```

Comment:
Not really a descriptive name.

Rename the test to something that describes it. For example "HasRole_Returns_Expected_Result".